### PR TITLE
[Refactor] Remove default_cluster from default_cluster:db

### PR DESCRIPTION
**Pull request by zombee0**
Signed-off-by: zombee0 <flylucas_10@163.com>

## What type of PR is this：
- [ ] bug
- [ ] feature
- [ ] enhancement
- [x] refactor
- [ ] others

## Which issues of this PR fixes ：

Fixes #

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 37 / 61 (60.66%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/http/meta/GlobalDictMetaService.java|0|1|00.00%|[83]|
|:large_blue_circle:|com/starrocks/service/FrontendServiceImpl.java|0|2|00.00%|[522, 564]|
|:large_blue_circle:|com/starrocks/mysql/privilege/UserProperty.java|0|1|00.00%|[263]|
|:large_blue_circle:|com/starrocks/persist/DatabaseInfo.java|0|6|00.00%|[81, 82, 83, 85, 95, 97]|
|:large_blue_circle:|com/starrocks/analysis/LabelName.java|0|2|00.00%|[104, 112]|
|:large_blue_circle:|com/starrocks/http/rest/ShowDataAction.java|0|1|00.00%|[75]|
|:large_blue_circle:|com/starrocks/persist/ModifyTableColumnOperationLog.java|0|2|00.00%|[29, 35]|
|:large_blue_circle:|com/starrocks/metric/MetricRepo.java|0|1|00.00%|[552]|
|:large_blue_circle:|com/starrocks/server/LocalMetastore.java|1|4|25.00%|[497, 624, 625]|
|:large_blue_circle:|com/starrocks/catalog/InfoSchemaDb.java|1|3|33.33%|[81, 88]|
|:large_blue_circle:|com/starrocks/analysis/TablePattern.java|2|4|50.00%|[134, 142]|
|:large_blue_circle:|com/starrocks/analysis/TableName.java|2|3|66.67%|[140]|
|:large_blue_circle:|com/starrocks/catalog/Database.java|6|6|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/parser/AstBuilder.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/scheduler/persist/TaskRunStatus.java|3|3|100.00%|[]|
|:large_blue_circle:|com/starrocks/cluster/Cluster.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/analyzer/InsertAnalyzer.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/backup/BackupHandler.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/statistic/StatisticUtils.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/persist/DropDbInfo.java|5|5|100.00%|[]|
|:large_blue_circle:|com/starrocks/persist/CreateTableInfo.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/server/GlobalStateMgr.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/analysis/FunctionName.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/scheduler/Task.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/mysql/privilege/DbPrivEntry.java|2|2|100.00%|[]|

**Comment by imay** [comment]
run starrocks_admit_test

**Comment by zombee0** [comment]
> run starrocks_admit_test

hi, I read the test's output, the reason may be it want to find db='default_cluster:%s' in the python program, but the default_cluster had been removed.

======================================================================
FAIL: test_resource_group.TestResourceGroup.test_resource_group_create
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/home/disk2/jenkins/workspace/starrocks_admit_test/StarRocksTest/case/system_case/test_ddl/test_resource_group/test_resource_group.py", line 134, in test_resource_group_create
    **eq_(res[0][9].find("db='default_cluster:%s'" % self.database_name) >= 0, True)**
AssertionError: False != True

**Comment by imay** [comment]
Okay, we need fix the test before merge it

**Comment by imay** [comment]
run starrocks_admit_test

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 37 / 61 (60.66%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/http/meta/GlobalDictMetaService.java|0|1|00.00%|[83]|
|:large_blue_circle:|com/starrocks/service/FrontendServiceImpl.java|0|2|00.00%|[522, 564]|
|:large_blue_circle:|com/starrocks/mysql/privilege/UserProperty.java|0|1|00.00%|[263]|
|:large_blue_circle:|com/starrocks/persist/DatabaseInfo.java|0|6|00.00%|[81, 82, 83, 85, 95, 97]|
|:large_blue_circle:|com/starrocks/analysis/LabelName.java|0|2|00.00%|[108, 116]|
|:large_blue_circle:|com/starrocks/http/rest/ShowDataAction.java|0|1|00.00%|[75]|
|:large_blue_circle:|com/starrocks/persist/ModifyTableColumnOperationLog.java|0|2|00.00%|[29, 35]|
|:large_blue_circle:|com/starrocks/metric/MetricRepo.java|0|1|00.00%|[552]|
|:large_blue_circle:|com/starrocks/server/LocalMetastore.java|1|4|25.00%|[501, 628, 629]|
|:large_blue_circle:|com/starrocks/catalog/InfoSchemaDb.java|1|3|33.33%|[81, 88]|
|:large_blue_circle:|com/starrocks/analysis/TablePattern.java|2|4|50.00%|[134, 142]|
|:large_blue_circle:|com/starrocks/analysis/TableName.java|2|3|66.67%|[140]|
|:large_blue_circle:|com/starrocks/catalog/Database.java|6|6|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/parser/AstBuilder.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/scheduler/persist/TaskRunStatus.java|3|3|100.00%|[]|
|:large_blue_circle:|com/starrocks/cluster/Cluster.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/analyzer/InsertAnalyzer.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/backup/BackupHandler.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/statistic/StatisticUtils.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/persist/DropDbInfo.java|5|5|100.00%|[]|
|:large_blue_circle:|com/starrocks/persist/CreateTableInfo.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/server/GlobalStateMgr.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/analysis/FunctionName.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/scheduler/Task.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/mysql/privilege/DbPrivEntry.java|2|2|100.00%|[]|

**Comment by zombee0** [comment]
run starrocks_admit_test

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 37 / 61 (60.66%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/http/meta/GlobalDictMetaService.java|0|1|00.00%|[83]|
|:large_blue_circle:|com/starrocks/service/FrontendServiceImpl.java|0|2|00.00%|[522, 564]|
|:large_blue_circle:|com/starrocks/mysql/privilege/UserProperty.java|0|1|00.00%|[263]|
|:large_blue_circle:|com/starrocks/persist/DatabaseInfo.java|0|6|00.00%|[81, 82, 83, 85, 95, 97]|
|:large_blue_circle:|com/starrocks/analysis/LabelName.java|0|2|00.00%|[108, 116]|
|:large_blue_circle:|com/starrocks/http/rest/ShowDataAction.java|0|1|00.00%|[75]|
|:large_blue_circle:|com/starrocks/persist/ModifyTableColumnOperationLog.java|0|2|00.00%|[29, 35]|
|:large_blue_circle:|com/starrocks/metric/MetricRepo.java|0|1|00.00%|[552]|
|:large_blue_circle:|com/starrocks/server/LocalMetastore.java|1|4|25.00%|[501, 628, 629]|
|:large_blue_circle:|com/starrocks/catalog/InfoSchemaDb.java|1|3|33.33%|[81, 88]|
|:large_blue_circle:|com/starrocks/analysis/TablePattern.java|2|4|50.00%|[134, 142]|
|:large_blue_circle:|com/starrocks/analysis/TableName.java|2|3|66.67%|[140]|
|:large_blue_circle:|com/starrocks/catalog/Database.java|6|6|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/parser/AstBuilder.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/scheduler/persist/TaskRunStatus.java|3|3|100.00%|[]|
|:large_blue_circle:|com/starrocks/cluster/Cluster.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/analyzer/InsertAnalyzer.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/backup/BackupHandler.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/statistic/StatisticUtils.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/persist/DropDbInfo.java|5|5|100.00%|[]|
|:large_blue_circle:|com/starrocks/persist/CreateTableInfo.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/server/GlobalStateMgr.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/analysis/FunctionName.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/scheduler/Task.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/mysql/privilege/DbPrivEntry.java|2|2|100.00%|[]|

**Comment by zombee0** [comment]
run starrocks_admit_test

**Comment by zombee0** [comment]
run starrocks_admit_test

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 37 / 61 (60.66%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/http/meta/GlobalDictMetaService.java|0|1|00.00%|[83]|
|:large_blue_circle:|com/starrocks/service/FrontendServiceImpl.java|0|2|00.00%|[522, 564]|
|:large_blue_circle:|com/starrocks/mysql/privilege/UserProperty.java|0|1|00.00%|[263]|
|:large_blue_circle:|com/starrocks/persist/DatabaseInfo.java|0|6|00.00%|[81, 82, 83, 85, 95, 97]|
|:large_blue_circle:|com/starrocks/analysis/LabelName.java|0|2|00.00%|[108, 116]|
|:large_blue_circle:|com/starrocks/http/rest/ShowDataAction.java|0|1|00.00%|[75]|
|:large_blue_circle:|com/starrocks/persist/ModifyTableColumnOperationLog.java|0|2|00.00%|[29, 35]|
|:large_blue_circle:|com/starrocks/metric/MetricRepo.java|0|1|00.00%|[552]|
|:large_blue_circle:|com/starrocks/server/LocalMetastore.java|1|4|25.00%|[496, 623, 624]|
|:large_blue_circle:|com/starrocks/catalog/InfoSchemaDb.java|1|3|33.33%|[81, 88]|
|:large_blue_circle:|com/starrocks/analysis/TablePattern.java|2|4|50.00%|[134, 142]|
|:large_blue_circle:|com/starrocks/analysis/TableName.java|2|3|66.67%|[140]|
|:large_blue_circle:|com/starrocks/catalog/Database.java|6|6|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/parser/AstBuilder.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/scheduler/persist/TaskRunStatus.java|3|3|100.00%|[]|
|:large_blue_circle:|com/starrocks/cluster/Cluster.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/analyzer/InsertAnalyzer.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/backup/BackupHandler.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/statistic/StatisticUtils.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/persist/DropDbInfo.java|5|5|100.00%|[]|
|:large_blue_circle:|com/starrocks/persist/CreateTableInfo.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/server/GlobalStateMgr.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/analysis/FunctionName.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/scheduler/Task.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/mysql/privilege/DbPrivEntry.java|2|2|100.00%|[]|

**Comment by zombee0** [comment]
run starrocks_admit_test

**Comment by zombee0** [comment]
run starrocks_admit_test

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 37 / 61 (60.66%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/http/meta/GlobalDictMetaService.java|0|1|00.00%|[83]|
|:large_blue_circle:|com/starrocks/service/FrontendServiceImpl.java|0|2|00.00%|[522, 564]|
|:large_blue_circle:|com/starrocks/mysql/privilege/UserProperty.java|0|1|00.00%|[263]|
|:large_blue_circle:|com/starrocks/persist/DatabaseInfo.java|0|6|00.00%|[81, 82, 83, 85, 95, 97]|
|:large_blue_circle:|com/starrocks/analysis/LabelName.java|0|2|00.00%|[108, 116]|
|:large_blue_circle:|com/starrocks/http/rest/ShowDataAction.java|0|1|00.00%|[75]|
|:large_blue_circle:|com/starrocks/persist/ModifyTableColumnOperationLog.java|0|2|00.00%|[29, 35]|
|:large_blue_circle:|com/starrocks/metric/MetricRepo.java|0|1|00.00%|[552]|
|:large_blue_circle:|com/starrocks/server/LocalMetastore.java|1|4|25.00%|[497, 624, 625]|
|:large_blue_circle:|com/starrocks/catalog/InfoSchemaDb.java|1|3|33.33%|[81, 88]|
|:large_blue_circle:|com/starrocks/analysis/TablePattern.java|2|4|50.00%|[134, 142]|
|:large_blue_circle:|com/starrocks/analysis/TableName.java|2|3|66.67%|[140]|
|:large_blue_circle:|com/starrocks/catalog/Database.java|6|6|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/parser/AstBuilder.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/scheduler/persist/TaskRunStatus.java|3|3|100.00%|[]|
|:large_blue_circle:|com/starrocks/cluster/Cluster.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/analyzer/InsertAnalyzer.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/backup/BackupHandler.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/statistic/StatisticUtils.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/persist/DropDbInfo.java|5|5|100.00%|[]|
|:large_blue_circle:|com/starrocks/persist/CreateTableInfo.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/server/GlobalStateMgr.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/analysis/FunctionName.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/scheduler/Task.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/mysql/privilege/DbPrivEntry.java|2|2|100.00%|[]|

**Comment by zombee0** [comment]
run starrocks_admit_test

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 37 / 61 (60.66%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/http/meta/GlobalDictMetaService.java|0|1|00.00%|[83]|
|:large_blue_circle:|com/starrocks/service/FrontendServiceImpl.java|0|2|00.00%|[522, 564]|
|:large_blue_circle:|com/starrocks/mysql/privilege/UserProperty.java|0|1|00.00%|[263]|
|:large_blue_circle:|com/starrocks/persist/DatabaseInfo.java|0|6|00.00%|[81, 82, 83, 85, 95, 97]|
|:large_blue_circle:|com/starrocks/analysis/LabelName.java|0|2|00.00%|[108, 116]|
|:large_blue_circle:|com/starrocks/http/rest/ShowDataAction.java|0|1|00.00%|[75]|
|:large_blue_circle:|com/starrocks/persist/ModifyTableColumnOperationLog.java|0|2|00.00%|[29, 35]|
|:large_blue_circle:|com/starrocks/metric/MetricRepo.java|0|1|00.00%|[552]|
|:large_blue_circle:|com/starrocks/server/LocalMetastore.java|1|4|25.00%|[497, 624, 625]|
|:large_blue_circle:|com/starrocks/catalog/InfoSchemaDb.java|1|3|33.33%|[81, 88]|
|:large_blue_circle:|com/starrocks/analysis/TablePattern.java|2|4|50.00%|[134, 142]|
|:large_blue_circle:|com/starrocks/analysis/TableName.java|2|3|66.67%|[140]|
|:large_blue_circle:|com/starrocks/catalog/Database.java|6|6|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/parser/AstBuilder.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/scheduler/persist/TaskRunStatus.java|3|3|100.00%|[]|
|:large_blue_circle:|com/starrocks/cluster/Cluster.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/analyzer/InsertAnalyzer.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/backup/BackupHandler.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/statistic/StatisticUtils.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/persist/DropDbInfo.java|5|5|100.00%|[]|
|:large_blue_circle:|com/starrocks/persist/CreateTableInfo.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/server/GlobalStateMgr.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/analysis/FunctionName.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/scheduler/Task.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/mysql/privilege/DbPrivEntry.java|2|2|100.00%|[]|

File: fe/fe-core/src/main/java/com/starrocks/service/FrontendServiceImpl.java [hunk 1]
```diff
@@ -519,7 +519,7 @@ public TGetDBPrivsResult getDBPrivs(TGetDBPrivsParams params) throws TException
             PrivBitSet savedPrivs = entry.getPrivSet();
             String clusterPrefix = SystemInfoService.DEFAULT_CLUSTER + ClusterNamespace.CLUSTER_DELIMITER;
```
Comment (chaoyli) [inline fe/fe-core/src/main/java/com/starrocks/service/FrontendServiceImpl.java:520]: I think we can remove the DEFAULT_CLUSTER also. The feature is useless but only has compatibility.

Comment (zombee0) [reply fe/fe-core/src/main/java/com/starrocks/service/FrontendServiceImpl.java:520]: thanks for review，up to now，we just remove default_cluster from default_cluster:db, and will remove default_cluster from default_cluster:role/user next step，so when default_cluster is used for role/user it is useful now

**Comment by gengjun-git** [comment]
@mergify backport branch-2.4

**Pull request merged**